app/src/components/Menu.css

```diff
@@ -110,4 +110,20 @@ ion-note {
 
```

Comment:
Should we be using SCSS and compiling this properly for the CDN so that we aren't doing CSS in JS. When we import stuff like graphs etc they will potentially have large css files so may be more of an issue on performance? Just something to think about? That being said CSS in JS is slightly more readable.

Reply:
Having said this I think I prefer proper CSS in JS like styled components for example (it works marginally better with my mindset cause css and js are in the same file). I cannot work out if its going to be an issue or not so thought I would bring it up before we went too deep into this so we can have the disccusion.

Reply:
@Jbithell Have you used Ionic before, do we have to have CSS in JS when its converted to android cause it cannot use the CDN at that point?

Reply:
cause I think it might have to be for the app sadly (which defeats the purpose of the cdn but hey)

Reply:
I can't remember the detail but it's probably worth considering that at compile time the compilation machine might not have internet access. I think we aim to bundle all assets if we can, also has the advantage of them being the right file type and size

Reply:
Gatsby handles this excellently but I'm sure there are equivalents for ionic

Reply:
https://ionicframework.com/docs/cli/commands/cordova-resources

Reply:
https://github.com/bstudios/adam-rms/issues/258